app/uk/gov/hmrc/agentservicesaccount/auth/SignOutUrl.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentservicesaccount.auth
+
+import javax.inject.{Inject, Singleton}
+
+import play.api.Configuration
+import uk.gov.hmrc.agentservicesaccount.config.RequiredConfigString
+import views.html.helper.urlEncode
+
+@Singleton
+class SignOutUrl @Inject() (override val configuration: Configuration) extends RequiredConfigString {
```

Comment:
yes let's do that (and rename to `ExternalUrls` or something because it won't just have sign out URLs in it any more)